src/css/popup-new.css

```diff
@@ -180,3 +186,112 @@ sign-up-panel::after {
   background-size: 120%;
 }
 
+/* Settings */
+
+.fx-relay-settings-toggles {
+  padding: var(--spacingMd);
+  background-color: var(--colorWhite);
+  border-radius: var(--borderRadiusSm);
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacingMd);
+}
+
+.fx-relay-settings-toggle-wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.fx-relay-settings-toggle-wrapper-spacer {
+  height: 1px;
+  background-color: var(--colorGrey10);
+}
```

Comment:
Woah, a spacer? Table layout flashbacks :sweat_smile: Any reason this couldn't be a padding, gap or border? (Preferably in a comment :) )

Reply:
<img width="387" alt="image" src="https://user-images.githubusercontent.com/2692333/214337038-66e250c3-e11f-439b-beda-206eddc63dce.png">

So my thought here was not to put padding on the child items (pink/green outlines), but rather use flexbox/gap on the parent. 

The alternate would be to drop the gap from the parent, and add `border-top` to one of the child items (dropping the spacer all together).

Reply:
Alright. On reflection, having unused markup HTML for CSS sake is bad practice.

Revised in 8b6669dabdd24d6443a837ac9a1288a3b2356f97

Preview: 
<img width="393" alt="image" src="https://user-images.githubusercontent.com/2692333/214340371-f118d9c7-632d-4004-a181-605a7d15eee6.png">